### PR TITLE
[NFV] Add the ovs-sriov scenario file

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: scenarios/reproducers/va-nfv-ovs-sriov.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+cifmw_architecture_scenario: "ovs-sriov"
+
+# Automation section. Most of those parameters will be passed to the
+# controller-0 as-is and be consumed by the `deploy-va.sh` script.
+# Please note, all paths are on the controller-0, meaning managed by the
+# Framework. Please do not edit them!
+_arch_repo: "/home/zuul/src/github.com/openstack-k8s-operators/architecture"
+
+# HERE if you want to overload kustomization, you can uncomment this parameter
```
Comment (hjensas) [inline scenarios/reproducers/va-nfv-ovs-sriov.yml:10]: nit: s/overload/override/ ?

**Review by hjensas — COMMENTED** [review]
/lgtm

**Comment by cjeanner** [comment]
/approve

new scenario, no impact on current CI content afaik.

**Comment by cjeanner** [comment]
/lgtm

re-adding lost LGTM upon nits corrections.

**Review by hjensas — COMMENTED** [review]
/lgtm

**Pull request merged**